submissions/adeyemiJuwon/database/transportationQuery.sql

```diff
@@ -0,0 +1,48 @@
+USE transportation;
+
+-- total number of passengers that survived
+SELECT COUNT(*) 
+FROM accidents 
+WHERE survived = 1;
+-- answer = 290
+
+-- total number of passengers that did not survive
+SELECT COUNT(*) 
+FROM accidents 
+WHERE survived = 0;
+-- answer = 424
```

Comment:
same here

Reply:
I guess you didn't import your data correctly. To even confirm, the total number of records on the CSV file for survival is `891` if you sum the two results it ought to give you that total (i.e 891). But 290 + 424 = 714 instead

Reply:
![import](https://user-images.githubusercontent.com/66218712/98976047-275fd700-2517-11eb-8e54-f1a67d2e255e.png)
I was only able to import 714 data and i have try it several time but still the same.
@adaugochi